defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-remi_repo_url: "http://rpms.remirepo.net/enterprise/remi-release-{{ ansible_distribution_major_version }}.rpm"
-remi_repo_gpg_key_url: "http://rpms.remirepo.net/RPM-GPG-KEY-remi"
+remi_repo_url: "https://mirror.us-midwest-1.nexcess.net/remi/enterprise/remi-release-{{ ansible_distribution_major_version }}.rpm"
+remi_repo_gpg_key_url: "https://mirror.us-midwest-1.nexcess.net/remi/enterprise/{{ ansible_distribution_major_version }}/RPM-GPG-KEY-remi"
```

Comment:
The larger change here is purposeful, actually reflects an upstream change that our fork is out of date about.